src/libraries/Microsoft.PowerFx.Core/Public/Config/ComposedReadOnlySymbolTable.cs

```diff
@@ -60,18 +59,25 @@ public override FormulaType GetTypeFromSlot(ISymbolSlot slot)
             return slot.Owner.GetTypeFromSlot(slot);
         }
 
+        private TexlFunctionSet _nameResolverFunctions = null;
+        private HashSet<VersionHash> _nameResolverFunctionsVersions = null;
```

Comment:
>private HashSet<VersionHash> _nameResolverFunctionsVersions = null; [](http://example.com/codeflow?start=8&length=67)

No - can we avoid this field?  And the SetEquals on 72. 

VersionHash (line 35) already provides a checksum